index.html

```diff
@@ -0,0 +1,233 @@
+                <div class="menu-desktop-top">
+                    <ul class="class-menu-desktop-top">
+                        <li>
+                            <a href="https://www.facebook.com/gilbert.mutai.14203544/">
+                                <img src="./img/facebook.png" alt="facebook">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="https://www.linkedin.com/in/mutai-gilbert-2a5a42137/">
+                                <img src="./img/Linkedin.png" alt="Linkedin">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">English</a>
+                        </li>
+                        <li>
+                            <a href="#">Mypage</a>
+                        </li>
+                        <li>
+                            <a href="#">Logout</a>
+                        </li>
+
+                    </ul>
+
+                </div>
+                <div class="menu-desktop">
+                    <div class="menu-atlas">
+                        <img src="./img/logo.png" alt="Menu Icon">
+
+                    </div>
+                    <ul class="menu-ul">
+                        <li class="menu-li">
+                            <a href="./about.html">About</a>
+                        </li>
+                        <li class="menu-li">Tickets</li>
+                        <li class="menu-li">Join</li>
+                        <li class="menu-li">Sponsor</li>
+                        <li class="menu-li">News</li>
+                        <li class="menu-li cc">CC Campaign</li>
+
+                    </ul>
+                </div>
+
+            </nav>
+        </div>
+
+    </header>
+    <main>
+        <section>
+            <div class="welcome">
+                <div class="intro">
+                    <h3> "Hello! Sharing World"</h3>
+                    <h1> "CREATIVE COMMONS <br> GLOBAL SUMMIT <br> 2015"</h1>
+                    <p>Praesent porttitor, mi quis pulvinar rutrum, nunc dolor rutrum enim, eu tempor nulla felis eu
+                        orci.
+                        Ut eu turpis nisi. Pellentesque convallis, purus vitae mollis ultrices, ex lectus consectetur
+                        nulla,
+                        et blandit ipsum nibh sit amet ipsum.</p>
+                    <h2> 2015.10.15(THU) ~ 16(FRI) <BR> @ National Museum of Korea, Art Center Nabi and More </h2>
+
+
+
+                </div>
+            </div>
+        </section>
+    </main>
+
+
+    <section id="about">
+        <div class="lets-grow">
+            <h3 class="lets-grow-title">Main Program</h3>
+            <ul class="programs">
+                <li class="Lecture">
+                    <img src="./img/Tools.png" alt="Tools">
+                    <h4>Lecture</h4>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
```

Comment:
As requested by the previous reviewers,  please 🙏🏽 use the correct spaces between the cards in the program section.

[Optional] I suggest that you remove the white borders from the cards and only add them as an effect on hover as shown by Cindy's design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/203924382-87d88912-01d1-44dc-ab96-e51eba0382ac.png)

Cindy's design:
![image](https://user-images.githubusercontent.com/20554583/203924438-82689a56-9315-423c-bcf8-c0901284b1f7.png)